src/client/src/Stream.c

```diff
@@ -1767,6 +1767,49 @@ STATUS streamFormatChanged(PKinesisVideoStream pKinesisVideoStream, UINT32 codec
     return retStatus;
 }
 
+/**
+ * Sets NALu adaptation flags.
+ */
+STATUS setNalAdaptionFlags(PKinesisVideoStream pKinesisVideoStream, UINT32 nalAdaptationFlags)
```

Comment:
should this say `setNalAdaptationFlags ` ?

Reply:
Yeah, let me replace for consistency